doc/dune-files.rst

```diff
@@ -285,6 +285,23 @@ language: The syntax is as a list of the following elements:
 
    dep-specification = dep+
 
+.. _always-add-cflags:
+
+future_c_and_cxx_flags_handling
+-----------------
+
+Since Dune 2.8, it is possible to deactivate the systematic prepending of flags
+coming from ``ocamlc -config`` to the C compiler command line. This is done
+adding the following field to the ``dune-project`` file:
+
+.. code:: scheme
+
+    (future_c_and_cxx_flags_handling true)
```

Comment:
I have to say that I don't really like this name. I'd prefer a name that describes that hints at what the feature is doing, rather than forecasting that it's going to be a default one day. What about something like `explicit_c_and_cxx_flags_handling` or `c_and_cxx_standard_flags_include_all`?

Reply:
Or maybe `use_standard_c_and_cxx_flags`?

Reply:
Sounds like a much better name to me.

Reply:
@jeremiedimino, @emillon, do you agree with this name `use_standard_c_and_cxx_flags` ?

If so I will rename every occurrence.

Reply:
I agree that the name is better, yes. I'd rather not use a boolean if possible, but this works too.

Reply:
> I agree that the name is better, yes. I'd rather not use a boolean if possible, but this works too.

I also much prefer `on`/`off` or `enabled`/`disabled`. I don't think we have any consistency on this stuff however.